extensions/cpp_region/PyRegion.cpp

```diff
@@ -425,8 +428,15 @@ void PyRegion::deserialize(BundleIO& bundle)
   
 }
 
+void PyRegion::write(capnp::AnyPointer::Builder& proto) const
+{
+  NTA_THROW << "Unimplemented method PyRegion::write.";
+}
 
-
+void PyRegion::read(capnp::AnyPointer::Reader& proto)
```

Comment:
No, the spec states that `override` should be used exactly once and I have to use it in the header.